frontend/src/admin/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from "react";
+import DropdownMenuIcon from "../dropdown-menu-icon/dropdown-menu-icon";
+import "./dropdown-menu.scss";
+
+interface DropdownMenuProps{
+    items: string[]
+    onClick: (value: string) => void
+}
+
+export default function DropdownMenu( {items, onClick}: DropdownMenuProps) {
+
+    const [showDropdown, setShowDropdown] = useState<boolean>(false);
```

Comment:
I am fairly certain the `<boolean>` is unnecessary, as you are giving it a default boolean value of `false`.

Reply:
✅ This is changed